tests/e2e/test_pgsql_node_reboot.py

```diff
@@ -0,0 +1,139 @@
+)
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.utility.spreadsheet.spreadsheet_api import GoogleSpreadSheetAPI
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def ripsaw(request):
+
+    # Create RipSaw Operator
+    ripsaw = RipSaw()
+
+    def teardown():
+        ripsaw.cleanup()
+    request.addfinalizer(teardown)
+    return ripsaw
+
+
+@ignore_leftovers
+@workloads
+@google_api_required
+@pytest.mark.polarion_id("OCS-806")
+class TestPgSQLNodeReboot(E2ETest):
+    """
+    Test running PGSQL and with Ceph pods respin
+    """
+    @pytest.fixture()
+    def pgsql_setup(self, ripsaw):
+        """
+        PGSQL test setup
+        """
+        # Deployment ripsaw and postgres database
+        log.info("Deploying postgres database")
+        ripsaw.apply_crd(
+            'resources/crds/'
+            'ripsaw_v1alpha1_ripsaw_crd.yaml'
+        )
+        ripsaw.setup_postgresql()
+
+        # Initialize Sanity instance
+        self.sanity_helpers = Sanity()
+
+    @pytest.mark.usefixtures(pgsql_setup.__name__)
+    def test_run_pgsql_node_reboot(self, transactions=900, node_type='master'):
+        """
+        Test pgsql workload
+        """
+        # Create pgbench benchmark
+        log.info("Create resource file for pgbench workload")
+        pg_trans = transactions
+        timeout = pg_trans * 3
+        pg_data = templating.load_yaml(constants.PGSQL_BENCHMARK_YAML)
+        pg_data['spec']['workload']['args']['transactions'] = pg_trans
+        pg_obj = OCS(**pg_data)
+        pg_obj.create()
+
+        # Wait for pgbench pod to be created
+        for pgbench_pod in TimeoutSampler(
+            pg_trans, 3, get_pod_name_by_pattern,
+            'pgbench', 'my-ripsaw'
+        ):
+            try:
+                if pgbench_pod[0] is not None:
+                    pgbench_client_pod = pgbench_pod[0]
+                    break
+            except IndexError:
+                log.info("Bench pod not ready yet")
+
+        # Node reboot with specific node type
+        typed_nodes = node.get_typed_nodes(node_type=node_type, num_of_nodes=1)
+        typed_node_name = typed_nodes[0].name
+
+        # Node maintenance - to gracefully terminate all pods on the node
+        node.drain_nodes([typed_node_name])
+
+        # Make the node schedulable again
+        node.schedule_nodes([typed_node_name])
+
+        # Perform cluster and Ceph health checks
+        self.sanity_helpers.health_check()
+
+        # Wait for pg_bench pod to complete workload
+        log.info("Waiting for pgbench_client to complete")
+        pod_obj = OCP(kind='pod')
+        pod_obj.wait_for_resource(
+            condition='Completed',
+            resource_name=pgbench_client_pod,
+            timeout=timeout,
+            sleep=10,
+        )
+
+        # Running pgbench and parsing logs
+        output = run_cmd(f'oc logs {pgbench_client_pod}')
```

Comment:
Can you please use get_pod_logs(podname, container_name) ?

Reply:
Same issue as get_pod_obj('pgbench_client_pod"), pgbench podname isn't changing with UUID postfix.